bessctl/commands.py

```diff
@@ -1190,6 +1195,10 @@ def show_mclass_list(cli, cls_names):
         _show_mclass(cli, cls_name, True)
 
 
+@cmd('import mclass LIB_FILE', 'Import the specified module library')
```

Comment:
We can envision that in the future we may be able to import multiple modules, drivers, hooks, scheduler policies, etc. from a single .so file package. In that sense, `import plugin PLUGIN_FILE` might be more general. Just `import PLUGIN_FILE` would also work, but there could be different types of "imports" to be added to bessctl (e.g., configuration)